gnocchi/tests/test_aggregates.py

```diff
@@ -64,12 +65,12 @@ def test_aggregated_different_archive_no_overlap(self):
                             self._resample_and_merge, agg_dict=tsc2))
 
         dtfrom = datetime64(2014, 1, 1, 11, 0, 0)
-        self.assertRaises(processor.UnAggregableTimeseries,
+        self.assertRaises(exceptions.UnAggregableTimeseries,
                           processor.aggregated,
                           [tsc1['return'], tsc2['return']],
                           from_timestamp=dtfrom,
                           operations=["aggregate", "mean", [
-                              "metric", ["foo", "mean"]]])
+                              "metric", ["all", "mean"]]])
```

Comment:
stupid question... what's 'all'? like literally every single metric in db? because that's insane.

Reply:
sorry, so this should not say 'all'?

Reply:
"all" is the default name we set for a metric for a metric in agg_name, see _resample_and_merge()

Reply:
word.